### PR TITLE
Adding Nvidia GPU Driver Installation to CentOS versions

### PR DESCRIPTION
**Pull request by anshuljainansja**

### CODE REVIEW

**Review by jithinjosepkl — COMMENTED** [review]
Can we add tests also?

File: common/install_nvidiagpudriver.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+set -ex
+
+# Install Cuda
+wget https://developer.download.nvidia.com/compute/cuda/11.0.3/local_installers/cuda_11.0.3_450.51.06_linux.run
+chmod +x cuda_11.0.3_450.51.06_linux.run
+sudo ./cuda_11.0.3_450.51.06_linux.run --silent
+echo 'export PATH=$PATH:/usr/local/cuda/bin' | sudo tee -a /etc/bash.bashrc
+echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64' | sudo tee -a /etc/bash.bashrc
+
+# Install DCGM
```
Comment (jithinjosepkl) [inline common/install_nvidiagpudriver.sh:11]: We can delete these commented lines (they wont work anyway for CentOS)

Comment (anshuljainansja) [reply common/install_nvidiagpudriver.sh]: Done

**Comment by anshuljainansja** [comment]
> Can we add tests also?

**Review by jithinjosepkl — APPROVED** [review]

**Pull request merged**